flocker/dockerplugin/test/test_api.py

```diff
@@ -174,6 +174,40 @@ def test_path(self):
                            u"Mountpoint": u"/flocker/{}".format(dataset_id)}))
         return d
 
+    def test_unknown_path(self):
+        """
+        ``/VolumeDriver.Path`` returns an error when asked for the mount path
+        of a non-existent volume.
+        """
+        name = u"myvol"
+        return self.assertResult(
+            b"POST", b"/VolumeDriver.Path",
+            {u"Name": name}, OK,
+            {u"Err": u"Volume not available.", u"Mountpoint": u""})
+
+    def test_non_local_path(self):
+        """
+        ``/VolumeDriver.Path`` returns an error when asked for the mount path
+        of a volume that is not mounted locally.
```

Comment:
...I'm assuming that the `VolumeDriver.Path` is only used when inspecting containers....I suppose it might also be used when docker stops a container and unmounts its volumes....or is there a separate `VolumeDriver.Unmount` endpoint?